internal/driver/webui_test.go

```diff
@@ -193,6 +194,10 @@ func makeFakeProfile() *profile.Profile {
 }
 
 func TestNewListenerAndURL(t *testing.T) {
+	if runtime.GOOS == "nacl" {
+		t.Skip("tcp Protocol not available on Nacl")
```

Comment:
Maybe use same message as in https://github.com/google/pprof/blob/9277d675724626f62433309e6dd3e8ec48a3e76e/internal/driver/fetch_test.go#L356?